pkg/cmd/images.go

```diff
@@ -0,0 +1,67 @@
+package cmd
+
+import (
+	"encoding/json"
+	"fmt"
+	"strings"
+
+	"github.com/openshift/microshift/pkg/release"
+	"github.com/spf13/cobra"
+	"sigs.k8s.io/yaml"
+
+	"k8s.io/cli-runtime/pkg/genericclioptions"
+	cmdutil "k8s.io/kubectl/pkg/cmd/util"
+)
+
+type imagesOptions struct {
+	Output string
+}
+
+func NewImagesCommand(ioStreams genericclioptions.IOStreams) *cobra.Command {
+	opts := imagesOptions{}
+
+	cmd := &cobra.Command{
+		Use:   "images",
+		Short: "Print container images used by this MicroShift version",
+		Run: func(cmd *cobra.Command, args []string) {
+			var marshalled []byte
+			var err error
+
+			switch opts.Output {
+			case "":
+				var buf strings.Builder
+				for _, v := range release.Image {
+					if buf.Len() > 0 {
+						fmt.Fprintf(&buf, " ")
+					}
+					fmt.Fprintf(&buf, v)
+				}
+				fmt.Fprintf(ioStreams.Out, "%s", buf.String())
```

Comment:
Is there any reason not to print directly to the stream in this case?

Reply:
Good point. FIxed.